weave-multicast/weave-multicast.md

```diff
@@ -0,0 +1,138 @@
+---
+
+
+Weave Net is one of the few Docker container networking solutions that supports multicast networking. Since Weave Net fully emulates a layer 2 network, it operates just like a server that is attached to a standard Ethernet network. Running multicast in Weave Net works even if the underlying network doesn’t support it, which is the case with Amazon Web Services (AWS) and Azure.
+
+With multicast, it is possible for a node to transmit its data once and then to have it delivered to multiple receivers in the same multicast group. Multicasting is used by software that generates and distributes data feeds to other applications, such as financial services.
+
+This tutorial runs a multicast demo in an ECS cluster created by an AWS CloudFormation template.  You will then visualize the end result in Weave Cloud. 
+
+The following topics are described: 
+
+* [What You Will Use](#what-use)
+* [Launching the AWS CloudFormation Template](#launching-AWS)
+* [Creating the Stack](#the-stack)
+* [Deploying the Multicast Demo to an EC2 Cluster](#multicast-demo)
+   * [Running Multicast Services in an ECS Cluster](#running-multicast)
+* [Visualizing Multicast in Weave Cloud](#visualize-multicast)
+
+##<a name="what-use"></a>What You Will Use
+
+* [Weave Net](https://www.weave.works/docs/net/latest/installing-weave/) and [Multicast](https://www.weave.works/docs/net/latest/features/)
+* AWS CloudFormation Template
+* EC2 Container Console
+* The [Weave Cloud](https://cloud.weave.works) token obtained after you've signed up. 
+
+##<a name="launching-AWS"></a>Launching the AWS CloudFomation Templates
+
+This AWS CloudFormation template is the easiest way to get started with Weave Net and Weave Cloud. CloudFormation templates provide a simple way to create a collection or a stack of related AWS resources, and allows you to provision and update them in an orderly and predictable fashion.
+
+Use this specially created Weaveworks CloudFormation template to create an EC2 cluster with all of the resources you need, including Weave Net and Weave Scope pre-installed onto an ECS Cluster.
+
+###Before You Begin
+
+Before launching the CloudFormation template:
+
+* [Set up an Amazon Account](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/get-set-up-for-amazon-ec2.html)
+* [Create the Key Pairs](http://docs.aws.amazon.com/gettingstarted/latest/wah/getting-started-prereq.html). You will need to reference the name of the key pairs when you create the stack.
+* Set up a [Weave Cloud](https://cloud.weave.works) account and [obtain the cloud token](https://www.weave.works/guides/using-weave-scope-cloud-service-to-visualize-and-monitor-docker-containers/).
+
+**Ready to launch a stack?  Click here to launch a stack to AWS:**
+
+[![](../images/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https:%2F%2Fs3.amazonaws.com%2Fweaveworks-cfn-public%2Fintegrations%2Fecs-baseline.json)
```

Comment:
We have just merged https://github.com/weaveworks/integrations/pull/101, so please update the URL to `https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https:%2F%2Fs3.amazonaws.com%2Fweaveworks-cfn-public%2Fintegrations%2Fecs-no-app.json`.